toolkit/docs/building/building.md

```diff
@@ -71,48 +76,101 @@ Quick Start
 ## Prerequisites
 Install prerequisites [here](prerequisites.md).
 
-## Build Tools
-By default all build commands are executed from inside the `./toolkit` folder.  A set of GO Based Tools and a Toolchain must be built before building Mariner Packages and Images.
+## Configure the Environment
+By default all build commands are executed from inside the `./toolkit` folder.
 
+A few one-time steps can be run now (they will re-run automatically later if needed)
+
+Remote files are generally only available for release branches.
 ```bash
-cd ~/git/CBL-Mariner/toolkit
-sudo make toolchain REBUILD_TOOLS=y REBUILD_TOOLCHAIN=y DOWNLOAD_SRPMS=y
+# Get the source code
+git clone https://github.com/microsoft/CBL-Mariner.git
+cd /CBL-Mariner/toolkit
```

Comment:
The cd should be relative instead of absolute.
```suggestion
cd CBL-Mariner/toolkit
```

Reply:
Thanks, updated